installer/templates/new/app.json

```diff
@@ -0,0 +1,19 @@
+{
+  "name": "Phoenix",
+  "description": "Elixir Web Framework targeting full-featured, fault tolerant applications with realtime functionality",
+  "website": "http://www.phoenixframework.org/",
+  <%= if ecto do %>"scripts": {
+    "postdeploy": "mix ecto.migrate"
+  },<% end %>
+  "env": {
+    "BUILDPACK_URL": "https://github.com/ddollar/heroku-buildpack-multi",
```

Comment:
I asked on heroku channel in freenode and a Heroku dev responded saying he had a pending PR he is waiting to be reviewed. Forgot his name though. :D So there's a pending PR but he said no definite date on when it will be shipped.